docs/python_docs/python/api/initializer/index.rst

```diff
@@ -0,0 +1,23 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+mxnet.initializer
+=================
+
+.. automodule:: mxnet.initializer
+    :members:
```

Comment:
Why no inherited-members?

Reply:
do you mean :imported-members:? if so then it's the same as : https://github.com/apache/incubator-mxnet/pull/16392#discussion_r332820265

if you meant :inherited-members: I think that only works for .. autoclass::

Reply:
> do you mean :imported-members:? if so then it's the same as : [#16392 (comment)](https://github.com/apache/incubator-mxnet/pull/16392#discussion_r332820265)
> 
> if you meant :inherited-members: I think that only works for .. autoclass::

I noticed that sometimes it's `imported-members` and sometimes it's `inherited-members`. I would like to better understand the distinction and when one should be used, both should be used, none should be used.